bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -105,6 +105,8 @@ contract NFTBridge is
 	uint256	_destinationChainId
   ) external whenNotPaused nonReentrant override {
     require(_msgSender() == federation, "NFTBridge: Not Federation");
+    checkChainId(_originChainId);
+		shouldBeCurrentChainId(_destinationChainId);
```

Comment:
Identation looks a bit weird in github, could you check that out please?, the same spacing difference happens on all the changes made

Reply:
yeah saw it, although it is ok in my IDE. I also fixed some other places where the identation was wrong.

Reply:
Ok I think I fixed it.